funcr/funcr.go

```diff
@@ -67,7 +67,7 @@ type Options struct {
 	// This has some overhead, so some users might not want it.
 	LogCaller MessageClass
 
-	// LogTimestamp tells funcr to add a "ts" key to log lines.  This has some
+	// LogTimestamps tells funcr to add a "ts" key to log lines.  This has some
```

Comment:
It's perhaps worth pointing out that this intentional revert to a typo is *not* flagged as an issue. My theory is that slight grammatical changes like appending "s" are not seen as a problem by the "revive" check.

Reply:
indeed.  It catches that same error on a public method.  We should file a bug against revive :)